microsoft-edge/webview2/release-notes/index.md

```diff
@@ -0,0 +1,1306 @@
+---
+title: Release Notes for the WebView2 SDK
+description: Release notes for Microsoft Edge WebView2, covering new features, APIs, and fixes for Win32, WPF, and WinForms.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.service: microsoft-edge
+ms.subservice: webview
+ms.date: 03/18/2024
+---
+# Release Notes for the WebView2 SDK
+
+<!-- covers ~5 months; periodically move oldest h2 sections from bottom of present file to archive.md.  eg covers:
+Feb 2024
+Jan 2024
+Dec 2023
+Nov 2023
+Oct 2023
+-->
+
+
```

Comment:
I was wondering if we should state at the top and link to about page and the archived page, similar to the Edge release notes for stable below:

```
These release notes provide information about new features and non-security updates that are included in the Microsoft Edge Stable Channel.

All the security updates are listed in [Release notes for Microsoft Edge Security Updates](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnotes-security).
Archived release notes for Microsoft Edge Stable Channel are located in [Archived release notes for Microsoft Edge Stable Channel](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-relnote-archive-stable-channel).
To understand Microsoft Edge channels, see the [Overview of the Microsoft Edge channels](https://learn.microsoft.com/en-us/deployedge/microsoft-edge-channels).
```

This is just in case people suddenly want to find the older stuff

Reply:
revising per other subseq. conversation/comment